zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -3289,8 +3293,6 @@ view.options.label.fontName					= Font Name:
 view.options.label.fontSize         		= Font Size:
 view.options.label.generalFont              = General Font
 view.options.label.workPanelsFont           = Work Panels Font
-view.options.label.largeRequestSize			= Large request view min size:
-view.options.label.largeResponseSize		= Large response view min size:
```

Comment:
What about the label for the view switching dropdown?

Reply:
OH I guess that stays until we actually remove the functionality.

Reply:
Right, that's in case some add-on is using the views (and thus the strings). Alternatively, I could hardcode the strings and remove them from the file.

Reply:
Naw, that’s just extra work. Thanks for tackling this. Seems to work wonderfully.